dora-team/dora.dev
# Fix: Adding Images to each Resource

√ Fix for https://github.com/dora-team/dora.dev/issues/101
Added images for each resource. 

<img width="1665" alt="Screenshot 2023-04-10 at 5 14 14 PM" src="https://user-images.githubusercontent.com/10970467/231023128-b6467c3b-9695-4235-b6d3-9a11c29a3936.png">

**Steps to Test:** 

- From the root directory pls run `hugo serve -D -s hugo --disableFastRender --verbose;`
- Check the changes at [http://localhost:1313/resources/](http://localhost:1313/resources/)
